scripts/after_prod.js

```diff
@@ -55,8 +55,5 @@ const cp = fs.copy;
       path.resolve(__dirname, '../packages/browser-extension/_locales'),
       path.resolve(__dirname, '../packages/browser-extension/build/_locales'),
     ),
-  ]).catch(err => {
-    console.log('failed to remove dev file in prod build');
-    throw err;
-  });
-})();
+  ]).catch(err => {}) // eslint-disable-line no-unused-vars
```

Comment:
Cause there's no dev file in prod build if running in CI. It will always log this error on CI.

This only happens on local machine, which won't be used in the publish workflow.